connector/producer.py

```diff
@@ -45,7 +46,7 @@
 @openerp.api.returns('self', lambda value: value.id)
 def create(self, vals):
     record_id = create_original(self, vals)
-    if self.pool.get('connector.installed') is not None:
+    if is_module_installed(self.env, 'connector') is not None:
```

Comment:
You, not see the good file.
At the top of the file you can read the import::
```python
     from .connector import is_module_installed
```

this function is an entry point, and this entry point is tested.